src/pipeline.js

```diff
@@ -90,53 +88,51 @@ class Pipeline {
 
     // function chain that was defined last. used for `when` and `unless`
     this._last = null;
-    // functions that are executed first
-    this._pres = [];
-    // function that is executed once
+    // function that is executed once and can register extensions
     this._oncef = null;
-    // functions that are executed after
-    this._posts = [];
+    // step functions to execute
+    this._steps = [];
     // functions that are executed before each step
     this._taps = [];
 
     this.attach = (ext) => {
```

Comment:
maybe _promote_ this to a proper method and add documentation?

Reply:
I don't understand: the method behind it is the new `use()`. I thought about renaming the property to `_use`, but found it a bit confusing.

Reply:
Oops, never mind: you meant `attach()` - of course!